go-fuzz-build/main.go

```diff
@@ -514,6 +512,13 @@ func (c *Context) buildInstrumentedBinary(blocks *[]CoverBlock, sonar *[]CoverBl
 	c.instrumentPackages(blocks, sonar)
 	mainPkg := c.createFuzzMain()
 	outf := c.tempFile()
+
+	cmdCleanImports := exec.Command("goimports", "-w", c.workdir)
```

Comment:
Not really important, but it is a good practice to explicitly tell the argument parser to stop parsing options when you pass in a variable based argument.

```suggestion
	cmdCleanImports := exec.Command("goimports", "-w", "--", c.workdir)
```